src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -90,21 +101,11 @@ private static void setupDrivers(Application application) {
         Job2dDriver driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
         DriverFeature.addDriver("Special line Simulator", driver);
 
-        driver = new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "special");
-        driver = new RotateTransformationDecorator(driver,45);
-        driver = new FlipTransformationDecorator(driver,true,false);
-        DriverFeature.addDriver("Rotated and flipped horizontally line Simulator", driver);
-
-        driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
-        driver = new ScaleTransformationDecorator(driver,2,2);
-        driver = new FlipTransformationDecorator(driver,false,true);
-        DriverFeature.addDriver("Scaled and flipped vertically special line Simulator", driver);
-
```

Comment:
When you add a new feature, you shouldn't remove existing ones! 
Please revert and redesign the original Transformation(Driver)Decorator (see PR #9) to work with PointTransformation (strategy).